.github/ISSUE_TEMPLATE/aws-access-request.yml

```diff
@@ -109,7 +109,10 @@ body:
   - type: textarea
     attributes:
       label: E-QIP Transmittal Confirmation
-      description: Attach a screenshot of your transmittal confirmation email by clicking this area to highlight and dragging the file in.
+      description: |
+        **NOTE: Do not submit PDF copies of your E-QIP document! These usually contain your social security number and other PII**
+
+        Your transmittal confirmation should contain the words "Transmittal Notice" in the subject of the email. If it doesn't, your E-QIP has not begun processing on the VA side.
       placeholder: Drag your transmittal confirmation email screenshot here
```

Comment:
Could we say something here that reiterates what the screenshot should be? 
<img width="297" alt="image" src="https://github.com/department-of-veterans-affairs/va.gov-team/assets/10993987/08bbc7ce-a3fb-4fb3-aca2-747a8e423096">

"Drag or paste your transmittal confirmation EMAIL screenshot here. Your transmittal confirmation should contain the words "Transmittal Notice" in the subject of the email. If it doesn't, your E-QIP has not begun processing on the VA side."

idk, that seems like a lot 😆